repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/AccessTypeRolesEntity.java

```diff
@@ -0,0 +1,210 @@
+package uk.gov.hmcts.ccd.definition.store.repository.entity;
+
+import javax.annotation.Nullable;
+import javax.persistence.Table;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.ManyToOne;
+import javax.persistence.JoinColumn;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+import static javax.persistence.FetchType.LAZY;
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Table(name = "access_type_roles")
+public class AccessTypeRolesEntity implements Serializable {
```

Comment:
```suggestion
@Entity
public class AccessTypeRolesEntity implements Serializable {
```

Please also add JPA `@Entity` annotation.